app/models/concerns/associations.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Associations
+  extend ActiveSupport::Concern
+
+  included do
+    class_attribute :associations, default: {}
+    class_attribute :has_one_associations, default: {}
+    class_attribute :has_many_associations, default: {}
+    class_attribute :belongs_to_associations, default: {}
+
+    class << self
+      # rubocop:disable Naming/PredicateName
+      def has_one(name, klass)
+        association(name, klass: klass, relation: :has_one, default: nil)
+      end
+      # rubocop:enable Naming/PredicateName
+
+      # rubocop:disable Naming/PredicateName
+      def has_many(name, klass)
+        association(name, klass: klass, relation: :has_many, default: [])
+      end
+      # rubocop:enable Naming/PredicateName
+
+      def belongs_to(name, klass)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/3bdd1603c87059b5dda5aabfb14999fb437bbf29/app/models/concerns/associations.rb#L25"></span>Takes parameters [‘klass’, ‘name’] to 4 methods</td>
    </tr>
  </tbody>
</table>